docs/root/source/index.rst

```diff
@@ -86,4 +86,3 @@ More About BigchainDB
    smart-contracts
    transaction-concepts
    timestamps
-   data-models/index
```

Comment:
Maybe leave the **Data Models** link for now (to help people who expect it to be there), but hard-wire it as a direct link over to the **Data Models** section in the server docs, i.e.
```text
   timestamps
   Data Models <https://docs.bigchaindb.com/projects/server/en/latest/data-models/index.html>
```